app/scripts/views/base.js

```diff
@@ -762,6 +762,13 @@ define(function (require, exports, module) {
       this._hasNavigated = true;
     },
 
+    navigateViaServer (url) {
```

Comment:
I did see the name `navigateToUrl` elsewhere and I'm not a fan. To me it fails to convey any meaningful difference to `navigate`. When viewed alongside each other, those two seem completely ambiguous I think.

I'm happy to drop `navigateViaServer` but I think the name(s) we use should reflect that the difference between the two methods is that one makes a request to a server and the other occurs within the client.

Thinking about alternative possibilities now.